u2f-ref-code/java/tests/com/google/u2f/key/messages/TransferAccessMessageTest.java

```diff
@@ -0,0 +1,255 @@
+
+    assertEquals(SEQUENCE_NUMBER, transferAccessMessage.getMessageSequenceNumber());
+    assertEquals(NEW_USER_PUBLIC_KEY, transferAccessMessage.getNewUserPublicKey());
+    assertArrayEquals(APPLICATION_SHA256, transferAccessMessage.getApplicationSha256());
+    assertEquals(NEW_ATTESTATION_CERTIFICATE, transferAccessMessage.getNewAttestationCertificate());
+    assertArrayEquals(SIGNATURE_USING_AUTHENTICATION_KEY,
+        transferAccessMessage.getSignatureUsingAuthenticationKey());
+    assertArrayEquals(SIGNATURE_USING_ATTESTATION_KEY,
+        transferAccessMessage.getSignatureUsingAttestationKey());
+
+  }
+
+  @Test
+  public void testTransferAccessMessage_FromBytes() throws U2FException {
+    TransferAccessMessage transferAccessMessage =
+        TransferAccessMessage.fromBytes(TRANSFER_ACCESS_MESSAGE_A_TO_B);
+
+    assertEquals(1, transferAccessMessage.getMessageSequenceNumber());
+    assertArrayEquals(TRANSFER_ACCESS_PUBLIC_KEY_B_HEX,
+        transferAccessMessage.getNewUserPublicKey());
+    assertArrayEquals(APP_ID_SIGN_SHA256, transferAccessMessage.getApplicationSha256());
+    assertEquals(VENDOR_CERTIFICATE, transferAccessMessage.getNewAttestationCertificate());
+    assertArrayEquals(TRANSFER_ACCESS_MESSAGE_SIGNATURE_USING_AUTHENTICATION_KEY_A_TO_B,
+        transferAccessMessage.getSignatureUsingAuthenticationKey());
+    assertArrayEquals(TRANSFER_ACCESS_MESSAGE_SIGNATURE_USING_ATTESTATION_KEY_A_TO_B,
+        transferAccessMessage.getSignatureUsingAttestationKey());
+    
+    assertTrue(crypto.verifySignature(transferAccessMessage.getNewAttestationCertificate(),
+        EXPECTED_TRANSFER_ACCESS_SIGNED_BYTES_FOR_ATTESTATION_KEY_A_TO_B,
+        transferAccessMessage.getSignatureUsingAttestationKey()));
+    assertTrue(crypto.verifySignature(USER_PUBLIC_KEY_SIGN,
+        EXPECTED_TRANSFER_ACCESS_SIGNED_BYTES_FOR_AUTHENTICATION_KEY_A_TO_B,
+        transferAccessMessage.getSignatureUsingAuthenticationKey()));
+  }
+
+  @Test
+  public void testTransferAccessMessage_FromBytes_2() throws U2FException {
+    TransferAccessMessage transferAccessMessage =
+        TransferAccessMessage.fromBytes(TRANSFER_ACCESS_MESSAGE_C_TO_D);
+
+    assertEquals(3, transferAccessMessage.getMessageSequenceNumber());
+    assertArrayEquals(TRANSFER_ACCESS_PUBLIC_KEY_D_HEX,
+        transferAccessMessage.getNewUserPublicKey());
+    assertArrayEquals(APP_ID_SIGN_SHA256, transferAccessMessage.getApplicationSha256());
+    assertEquals(VENDOR_CERTIFICATE, transferAccessMessage.getNewAttestationCertificate());
+    assertArrayEquals(TRANSFER_ACCESS_MESSAGE_SIGNATURE_USING_AUTHENTICATION_KEY_C_TO_D,
+        transferAccessMessage.getSignatureUsingAuthenticationKey());
+    assertArrayEquals(TRANSFER_ACCESS_MESSAGE_SIGNATURE_USING_ATTESTATION_KEY_C_TO_D,
+        transferAccessMessage.getSignatureUsingAttestationKey());
+  }
+
+  @Test
+  public void testTransferAccessMessage_FromBytes_ExtraBytes() {
+    try {
+      @SuppressWarnings("unused")
```

Comment:
don't suppress the warning, just don't assign the result of the `fromBytes()` call.